tests/Unit/FilesystemTestCase.php

```diff
@@ -6,44 +6,71 @@
 use WPMedia\PHPUnit\Unit\VirtualFilesystemTestCase;
 
 abstract class FilesystemTestCase extends VirtualFilesystemTestCase {
-	protected $wprocket_structure = [
-		'example.org'                             => [
-			'index.html'      => '',
-			'index.html_gzip' => '',
-			'about'           => [
-				'index.html'             => '',
-				'index.html_gzip'        => '',
-				'index-mobile.html'      => '',
-				'index-mobile.html_gzip' => '',
-			],
-			'category'        => [
-				'wordpress' => [
+	protected $structure = [
+		'wp-rocket' => [
+			'example.org'                             => [
+				'index.html'      => '',
+				'index.html_gzip' => '',
+				'about'           => [
+					'index.html'             => '',
+					'index.html_gzip'        => '',
+					'index-mobile.html'      => '',
+					'index-mobile.html_gzip' => '',
+				],
+				'category'        => [
+					'wordpress' => [
+						'index.html'      => '',
+						'index.html_gzip' => '',
+					],
+				],
+				'blog'            => [
+					'index.html'      => '',
+					'index.html_gzip' => '',
+				],
+				'en'              => [
 					'index.html'      => '',
 					'index.html_gzip' => '',
 				],
 			],
-			'blog'            => [
-				'index.html'      => '',
-				'index.html_gzip' => '',
-			],
-			'en'              => [
+			'example.org-Greg-594d03f6ae698691165999' => [
 				'index.html'      => '',
 				'index.html_gzip' => '',
 			],
 		],
-		'example.org-Greg-594d03f6ae698691165999' => [
-			'index.html'      => '',
-			'index.html_gzip' => '',
+		'min' => [
+			'1' => [
+				'5c795b0e3a1884eec34a989485f863ff.js'     => '',
+				'5c795b0e3a1884eec34a989485f863ff.js.gz'  => '',
+				'fa2965d41f1515951de523cecb81f85e.css'    => '',
+				'fa2965d41f1515951de523cecb81f85e.css.gz' => '',
+			],
 		],
+		'busting' => [
+			'1' => [
+				'wp-content' => [
+					'themes' => [
+						'storefront' => [
+							'assets' => [
+								'js' => [
+									'navigation.min-2.5.3.js'    => '',
+									'navigation.min-2.5.3.js.gz' => '',
+								],
+							],
+							'style-2.5.3.css'    => '',
+							'style-2.5.3.css.gz' => '',
+						],
+					],
+				],
+			]
+		]
 	];
 	protected $cache_path;
 
 	public function setUp() {
-		$this->structure['wp-rocket'] = $this->wprocket_structure;
 		parent::setUp();
 
 		// Redefine rocket_direct_filesystem() to use the virtual filesystem.
 		Functions\when( 'rocket_direct_filesystem' )->justReturn( $this->filesystem );
-		$this->cache_path = $this->filesystem->getUrl( 'wp-rocket' );
+		$this->cache_path = $this->filesystem->getUrl( 'cache' );
```

Comment:
@Tabrisrp This isn't be used anywhere in the tests. Is it needed?

Reply:
Currently no, this was in the file you initially wrote so I kept it

Reply:
Let's go ahead and delete this. I did add it to the `wp-media/phpunit` base test case, just in case we ever need it. https://github.com/wp-media/phpunit/pull/3